pipeline_steps/github.groovy

```diff
@@ -71,4 +71,31 @@ void add_issue_url_to_pr(){
 }
 
 
+def merge_pr(
+    String repo,
+    String pr_number,
+    String retries=5,
+    String org="rcbops"
```

Comment:
Could move `org` to first arg to keep associated args together, but complete nit.